docs/src/main/tut/json.md

```diff
@@ -35,14 +35,14 @@ libraryDependencies += "org.http4s" %% "http4s-json4s-jackson" % "0.15.0a-SNAPSH
 
 ## Import it
 
-The namespace is
+The import statement is one of the following:
 
 ```scala
-import org.http4s.<libraryName>._
+import org.http4s.argonaut._
+import org.http4s.circe._
+import org.http4s.json4s._
```

Comment:
It's `org.http4s.json4s.jackson._` or `org.http4s.json4s.native._`.  This one is ... special.